src/pages/dataTableUtilities/reducer/rowReducers.js

```diff
@@ -33,10 +33,12 @@ export const deselectRow = (state, action) => {
   const rowState = newDataState.get(rowKey);
   newDataState.set(rowKey, { ...rowState, isSelected: false });
 
+  const hasSelection = Array.from(newDataState).some(([, { isSelected }]) => isSelected);
```

Comment:
NIT: I think can be made a bit more readable by using `.values()` and spitting into two lines (untested):

```
const dataStateValues = Array.from(newDataState.values());
const hasSelected = dataStateValues.some(({ isSelected }) => isSelected);
```
```

Reply:
87gf10z